includes/admin/class-sensei-learners-admin-bulk-actions-view.php

```diff
@@ -259,10 +363,20 @@ public function data_table_header() {
 		<?php
 	}
 
-	public function search_button( $text = '' ) {
+	/**
+	 * Returns the search button text.
+	 */
+	public function search_button() {
 		return __( 'Search Learners', 'sensei-lms' );
 	}
 
+	/**
+	 * Helper method to display the content of the overview column of the table.
```

Comment:
I wonder if it is confusing that these numbers only reflect progress and not enrolment. Should we add something for enrolment?

Reply:
Added an enrolments column in this commit bf58b766472218e5e910bb4f26f80f8b0a26e6c0
I also improved the behavior of the overview column as discussed in fcd96b49639f82cf44f7abb85b90eaa6b9faaebc